### PR TITLE
feat(input-range): create input-range component

### PR DESCRIPTION
**Pull request by erikkroes**

### CODE REVIEW

File: packages/input-range/stories/index.stories.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+        min="0"
+        max="100"
+        modelValue="50"
+        unit="%"
+        label="Input range"
+      ></lion-input-range>
+    `,
+  )
+  .add(
+    'Help text',
```
Comment (gerjanvangeest) [inline packages/input-range/stories/index.stories.js]: You demo this twice `l30`

File: packages/input-range/test/lion-input-range.test.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+  it('is accessible', async () => {
+    const el = await fixture(`<lion-input-range label="range"></lion-input-range>`);
+    await expect(el).to.be.accessible();
+  });
+
+  it('is accessible when disabled', async () => {
+    const el = await fixture(`<lion-input-range label="range" disabled></lion-input-range>`);
+    await expect(el).to.be.accessible();
+  });
```
Comment (gerjanvangeest) [inline packages/input-range/test/lion-input-range.test.js]: Add a11y audit test

File: packages/input-range/src/LionInputRange.js [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+  connectedCallback() {
+    if (super.connectedCallback) super.connectedCallback();
+    this.type = 'range';
+  }
+
+  constructor() {
+    super();
+    this.parser = value => parseFloat(value);
+    if (this.min && this.max) {
+      this.defaultValidators.push(new MinMaxNumber({ min: this.min, max: this.max }));
```
Comment (gerjanvangeest) [inline packages/input-range/src/LionInputRange.js]: It is only possible to get wrong values when the modelValue is set programmatically. Don't think this is needed.

File: packages/input-range/src/LionInputRange.js [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+  static get properties() {
+    return {
+      min: Number,
+      max: Number,
+      unit: String,
+      step: {
+        type: Number,
+        reflect: true,
+      },
+      ticks: {
```
Comment (gerjanvangeest) [inline packages/input-range/src/LionInputRange.js]: This is not yet implemented, please remove.

File: packages/input-range/src/LionInputRange.js [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+
+    if (changedProperties.has('modelValue')) {
+      this._inputNode.value = this.modelValue;
+    }
+
+    if (changedProperties.has('step')) {
+      this._inputNode.step = this.step;
+    }
+
+    if (changedProperties.has('ticks')) {
```
Comment (gerjanvangeest) [inline packages/input-range/src/LionInputRange.js]: Not yet implemented, please remove.

File: packages/input-range/test/lion-input-range.test.js [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+  it('update min and max attributes when min and max property change', async () => {
+    const el = await fixture(`<lion-input-range min="100" max="200"></lion-input-range>`);
+    el.min = '120';
+    el.max = '220';
+    await nextFrame();
+    expect(el._inputNode.min).to.equal(el.min);
+    expect(el._inputNode.max).to.equal(el.max);
+  });
+
+  it.skip('can change the amount of tick labels', async () => {
```
Comment (gerjanvangeest) [inline packages/input-range/test/lion-input-range.test.js]: Please clean up

File: packages/input-range/test/lion-input-range.test.js [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+
+  it('displays the modelValue and unit', async () => {
+    const el = await fixture(`
+      <lion-input-range modelValue="${75}" unit="${`%`}"></lion-input-range>
+    `);
+    expect(el.shadowRoot.querySelector('.input-range__value').innerText).to.equal('75');
+    expect(el.shadowRoot.querySelector('.input-range__unit').innerText).to.equal('%');
+  });
+
+  // todo: define what to display
```
Comment (gerjanvangeest) [inline packages/input-range/test/lion-input-range.test.js]: Check this `todo`

File: packages/input-range/test/lion-input-range.test.js [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+  it('update min and max attributes when min and max property change', async () => {
+    const el = await fixture(`<lion-input-range min="100" max="200"></lion-input-range>`);
+    el.min = '120';
+    el.max = '220';
+    await nextFrame();
+    expect(el._inputNode.min).to.equal(el.min);
+    expect(el._inputNode.max).to.equal(el.max);
+  });
+
+  it('can hide the tick labels', async () => {
```
Comment (gerjanvangeest) [inline packages/input-range/test/lion-input-range.test.js]: This one is not yet implemented

Comment (gerjanvangeest) [reply packages/input-range/test/lion-input-range.test.js]: Change to `noRangeLabels`?

File: packages/input-range/README.md [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+```
+
+```js
+import '@lion/input-range/lion-input-range.js';
+```
+
+### Example
+
+```html
+<lion-input-range min="200" max="500" modelValue="300" label="Input range"></lion-input-range>
```
Comment (daKmoR) [inline packages/input-range/README.md]:
```suggestion
<lion-input-range min="200" max="500" .modelValue=${300} label="Input range"></lion-input-range>
```

File: packages/input-range/src/LionInputRange.js [hunk 10]
```diff
@@ -0,0 +1,92 @@
...
+import { LionInput } from '@lion/input';
+import { html } from '@lion/core';
+
+/**
+ * LionInputRange: extension of lion-input
+ *
+ * @customElement `lion-input-range`
+ * @extends LionInput
+ */
+export class LionInputRange extends FieldCustomMixin(LocalizeMixin(LionInput)) {
```
Comment (daKmoR) [inline packages/input-range/src/LionInputRange.js]: why LocalizeMixin?

**Pull request merged**